lib/ansible/modules/cloud/misc/virt.py

```diff
@@ -450,6 +461,11 @@ def core(module):
             module.fail_json(msg = "state change requires a guest specified")
 
         res['changed'] = False
```

Comment:
autostart is valid when VM will exists after module run... hmm what is the best option